### PR TITLE
Use our own ansible.cfg rather than the default CI slave.

### LINKED ISSUE #56: Remove the deprecated accelerate as a connection method
doing so will get rid of the deprecation warnings 

```
[DEPRECATION WARNING]: ACCELERATE_TIMEOUT option, Removing accelerate as a 
connection method, settings not needed either. . This feature will be removed 
in version 2.5. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.

```

### PR DESCRIPTION
**Pull request by DanyC97**
Fix #56 

Also done a bit of tidy up on the JenkinsFile

- [x] added `PYTHONUNBUFFERED` to turn off stdout buffering so that I can see the next output line immediately in the Jenkins console view
- [x] removed host_key check and connection since that is covered by the `ansible.cfg`
- [x] removed the deprecated `accelerate_*`

### CODE REVIEW

**Comment by DanyC97** [comment]
@tdawson @arilivigni if you don't mind reviewing please.

File: Jenkinsfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
-env.ANSIBLE_SSH_CONTROL_PATH = "%(directory)s/%%h-%%r"
-env.ANSIBLE_HOST_KEY_CHECKING = false
+env.PYTHONUNBUFFERED = 1
 env.ANSIBLE_FORCE_COLOR = true
```
Comment (arilivigni) [inline Jenkinsfile]: Can we move this into ansible.cfg file as well

Comment (DanyC97) [reply Jenkinsfile]: i tried but i didn't get the expected result while tested

**Comment by DanyC97** [comment]
and one important missing feature ;)
By having an updated `ansible.cfg` which we can change how we want w/o asking for root permission on CI slave, we now have a nice summary of each task and so we know where the time is being spent ;)

```
17:45:14 Tuesday 10 April 2018  17:45:08 +0000 (0:00:00.732)       0:03:26.402 ********* 
17:45:14 =============================================================================== 
17:45:14 cbs : Cbs build target for openshift-ansible from srpm ---------------- 178.32s
17:45:14 /home/sig-paas/workspace/paas-sig-ci-dev/paas-ci/playbooks/openshift/roles/cbs/tasks/cbs_build_srpms.yml:28 
17:45:14 cbs : Build srpm with tito for openshift-ansible ------------------------ 6.63s
17:45:14 /home/sig-paas/workspace/paas-sig-ci-dev/paas-ci/playbooks/openshift/roles/cbs/tasks/cbs_build_srpms.yml:3 
17:45:14 common : Update all packages -------------------------------------------- 4.51s
17:45:14 /home/sig-paas/workspace/paas-sig-ci-dev/paas-ci/playbooks/openshift/roles/common/tasks/install_required_pkgs.yml:5 
17:45:14 cbs : Install packages on repo_host in group_vars for cbs --------------- 3.95s
17:45:14 /home/sig-paas/workspace/paas-sig-ci-dev/paas-ci/playbooks/openshift/roles/cbs/tasks/install_cbs_pkgs.yml:3 
17:45:14 cbs : Install certs on repo_host in group_vars for cbs ------------------ 3.28s
17:45:14 /home/sig-paas/workspace/paas-sig-ci-dev/paas-ci/playbooks/openshift/roles/cbs/tasks/copy_cbs_certs.yml:3 
17:45:14 common : Get epel RPM signing key --------------------------------------- 2.18s

```

File: Jenkinsfile (hunk 1 above)
Comment (DanyC97) [reply Jenkinsfile]: trying again ...

Comment (arilivigni) [reply Jenkinsfile]: bitmoji

**Comment by DanyC97** [comment]
fyi @arilivigni your wish came true :) one more variable gone from jenkinsFile

**Pull request merged**